shm/.gitignore

```diff
@@ -0,0 +1 @@
+notes.txt
```

Comment:
There is a .gitignore one directory up (above shm) - plase append that line there.